### PR TITLE
Add code and circuit description

### PR DESCRIPTION
**Pull request by GRINGOLOCO7**
added code for minotauro

### CODE REVIEW

**Comment by GRINGOLOCO7** [comment]
@dsanmart can you review this?

File: README.md [hunk 1]
```diff
@@ -12,6 +12,23 @@ Minotauro is a defensive combat robot designed to compete in the Oshwdem 2023 co
 
 # Circuit
 
+Diagram:
+
+![Alt text](images/ps3_controlled_car.png)
```
Comment (dsanmart) [inline README.md]: Restricting the height is usually a good practice for controlling the size of the image. For example:

```
<div style="text-align:center"><img src="./images/ps3_controlled_car.png" width="" height="150" /></div>
```

Comment (dsanmart) [inline README.md]: In between [] goes the description of the image. Replace "Alt text" for "Circuit Diagram"

File: README.md [hunk 2]
```diff
@@ -12,6 +12,23 @@ Minotauro is a defensive combat robot designed to compete in the Oshwdem 2023 co
 
 # Circuit
 
+Diagram:
```
Comment (dsanmart) [inline README.md]: This could go in the title:

`# Circuit Diagram`

File: README.md [hunk 3]
```diff
@@ -12,6 +12,23 @@ Minotauro is a defensive combat robot designed to compete in the Oshwdem 2023 co
...
+
+![Alt text](images/ps3_controlled_car.png)
+
+Components:
```
Comment (dsanmart) [inline README.md]: Make this a title: `Robot Components`

File: README.md [hunk 4]
```diff
@@ -12,6 +12,23 @@ Minotauro is a defensive combat robot designed to compete in the Oshwdem 2023 co
...
+
+- **Lithium-Ion Batteries**: Two lithium-Ion Batteries 3.7V. The batteries are connected in series to reach a voltage of 7V. They are connected to a voltage regulator.
+- **voltage sensor regulator**: Used to decrease 7.2V to 6V (Maximum voltage accepted by DCmotors). Using a potentiometer we adjisted the voltage to the desired one and pass to the L293N Motor Driver.
+- **L293N Motor Driver**: This components allow us to regulate the speed and senso of rotation of the wheels. The driver recive power and supply it to 4 DC motor and the esp32 board. 
+- **ESP32 Board**: Recive power to the Motor Driver and output the information receaved by the PS4 controller joistick. 
+
+- **PS4 Contrller**: Connected via Bluetooth to the esp32 driver register and send as output the information recived by left&right joystick.
+-**DC Motor**: 4 Dc motor connected in parallel 2 (left) by 2 (right).
+
+code source: [here](https://github.com/un0038998/PS4Controller_Tank)
```
Comment (dsanmart) [inline README.md]: This should be referenced in the code file (at the begining) instead of in the README

File: code/CODE.ino [hunk 5]
```diff
@@ -0,0 +1,113 @@
+#include <PS4Controller.h>
```
Comment (dsanmart) [inline code/CODE.ino:3]: First line comment should say something like:

`// code source: https://github.com/un0038998/PS4Controller_Tank`

File: images/ps3_controlled_car.png
Comment (dsanmart) [inline images/ps3_controlled_car.png:1]: Please add a descriptive name for the image. E.g: `circuit_diagram.png`

File: README.md [hunk 6]
```diff
@@ -12,6 +12,23 @@ Minotauro is a defensive combat robot designed to compete in the Oshwdem 2023 co
...
+
+diagram source: [here](https://github.com/un0038998/PS3ControlledCar/blob/main/Diagrams/PS3%20Controlled%20Car.png)
```
Comment (dsanmart) [inline README.md]: This should go in the diagram section above

**Review by dsanmart — CHANGES_REQUESTED** [review]
Minor fixes

**Review by dsanmart — APPROVED** [review]
lgtm

**Pull request merged**